lib/dependencies/HarmonyEvaluatedImportSpecifierDependency.js

```diff
@@ -61,14 +61,43 @@ HarmonyEvaluatedImportSpecifierDependency.Template = class HarmonyEvaluatedImpor
 		const dep = /** @type {HarmonyEvaluatedImportSpecifierDependency} */ (
 			dependency
 		);
-		const { moduleGraph, runtime } = templateContext;
+		const { module, moduleGraph, runtime } = templateContext;
 		const connection = moduleGraph.getConnection(dep);
 		// Skip rendering depending when dependency is conditional
 		if (connection && !connection.isTargetActive(runtime)) return;
 
 		const exportsInfo = moduleGraph.getExportsInfo(connection.module);
 		const ids = dep.getIds(moduleGraph);
-		const value = exportsInfo.isExportProvided(ids);
+
+		let value;
+
+		const exportsType = connection.module.getExportsType(
+			moduleGraph,
+			module.buildMeta.strictHarmonyModule
+		);
+		switch (exportsType) {
+			case "default-with-named": {
+				value = exportsInfo.isExportProvided(
+					ids[0] === "default" ? ids.slice(1) : ids
```

Comment:
This could end up with `[]` which `isExportProvided` doesn't like.
I think we can use `true` for that case

Reply:
`"default" in cjsModule`

Reply:
yeah, good catch

Reply:
hm.. `"default" in cjsModule` will have `["default", "default"]` ids, not sure that with `HarmonyEvaluatedImportSpecifierDependency.js` this possible

Reply:
anyway I fixed it